Schema/PublicSchema/ContextTagKeys.bond

```diff
@@ -0,0 +1,105 @@
+
+namespace AI
+
+[ContextContract("Emit")]
+[PseudoType("JSMap")]
+struct ContextTagKeys
+{
+    [Description("Application version. Information in the application context fields is always about the application that is sending the telemetry.")]
+    [MaxStringLength("1024")]
+    10: string 	 ApplicationVersion = "ai.application.ver";
+    
+    [Description("Unique client device id. Computer name in most cases.")]
+    [MaxStringLength("1024")]
+    100: string 	 DeviceId = "ai.device.id";
+    
+    [Description("Device locale using <language>-<REGION> pattern, following RFC 5646. Example 'en-US'.")]
+    [MaxStringLength("64")]
+    115: string 	 DeviceLocale = "ai.device.locale";
+    
+    [Description("Model of the device the end user of the application is using. Used for client scenarios. If this field is empty then it is derived from the user agent.")]
+    [MaxStringLength("256")]
+    120: string 	 DeviceModel = "ai.device.model";
+    
+    [Description("Client device OEM name taken from the browser.")]
+    [MaxStringLength("256")]
+    130: string 	 DeviceOEMName = "ai.device.oemName";
+    
+    [Description("Operating system name and version of the device the end user of the application is using. If this field is empty then it is derived from the user agent. Example 'Windows 10 Pro 10.0.10586.0'")]
+    [MaxStringLength("256")]
+    140: string 	 DeviceOSVersion = "ai.device.osVersion";
+    
+    [Description("The type of the device the end user of the application is using. Used primarily to distinguish JavaScript telemetry from server side telemetry. Examples: 'PC', 'Phone', 'Browser'. 'PC' is the default value.")]
+    [MaxStringLength("64")]
+    160: string 	 DeviceType = "ai.device.type";
+    
+    [Description("The IP address of the client device. IPv4 and IPv6 is supported. Information in the location context fields is always about the end user. When telemetry is sent from a service, the location context is about the user that initiated the operation in the service.")]
+    [MaxStringLength("45")]
```

Comment:
Will there be a trailing null terminator that would make this need to be 46 instead of 45?